### ISSUE TITLE
Junction window transparent and hard to see

### ISSUE THREAD

**Issue by chase9** [sonnyp/Junction#70]
I have an issue using Fedora 35, KDE, and Junction in a Flatpak. The dialog opens as shown below, and can be very hard to see depending on what's behind it.

![image](https://user-images.githubusercontent.com/6013163/162536493-d5e569ae-f893-44c7-97b3-b75cd18d1a02.png)

It seems this always happens, except for when I click on a link from another Flatpak application, in which case the option for Edge is missing but the window is fully opaque.

**Comment by sonnyp** [comment]
I don't really know what I can do about this. Could you report the issue on the KWin bug tracker?

**Comment by chase9** [comment]
Ok, I've started a bug with KDE to get their feedback:
https://bugs.kde.org/show_bug.cgi?id=455251

**Comment by chase9** [comment]
I'm _extremely_ embarrassed to say that it appears the issue was caused by myself. At some point, I was building Junction myself and must have set the opacity property to .1 for window.ui. I had thought I wiped my own build of Junction from my system but apparently I did not.

Launching Junction from other flatpak'd applications apparently prefers using flatpak, which is why the problem didn't appear. Sorry for wasting your time!
